pkg/status/status-controller.go

```diff
@@ -381,7 +384,12 @@ func shouldSkipUpdate(old, new interface{}) bool {
 	return newMObj.GetResourceVersion() == oldMObj.GetResourceVersion()
 }
 
-func shouldSkipDelete(_ interface{}) bool {
+func objNotInThisWDS(obj interface{}, thisWDS string) bool {
```

Comment:
This object is not going to be **in** the WDS, it might be **for** or **from** the WDS.